projects/laji/src/app/shared-modules/pill-list/pill-list.component.scss

```diff
@@ -0,0 +1,26 @@
+@import 'vars';
+
+.label {
+  display: inline-block;
+  margin-top: $sp-1;
+  margin-right: $sp-1;
+
+  font-size: $fs-1;
+  color: $success-1;
+  background-color: $success-5;
+
+  box-shadow: 0 1px 2px 0 rgba(#000, .2);
```

Comment:
I'm wondering if we should also add a global class like `elevation-1`, `elevation-2` etc so that we wouldn't end up using different styles in different places

Reply:
I've also thought about this, but there's a problem with it: Box-shadow can't be broken down into its component properties. This would be required, because elements with a strong background-color should have colored box-shadows (ironically I forgot to do it here, fixing it).

Reply:
ok yes first comment was based on the black border color and if the color needs to be alternated then having global class would become a problematic :) Mixing maybe... end goal would be to find some good way to keep the elevation effects uniform.